Makefile

```diff
@@ -187,7 +188,7 @@ bin/cri-integration.test:
 
 cri-integration: binaries bin/cri-integration.test ## run cri integration tests
 	@echo "$(WHALE) $@"
-	@./script/test/cri-integration.sh
+	@bash -x ./script/test/cri-integration.sh
```

Comment:
Windows doesn't honour `#!`, so you need to explicitly run `bash`. This will require bash to be installed and in your `PATH`, but if you're running a Makefile, there's a good chance that's either the case, or not hard to arrange, depending on how you sourced `make`.

Reply:
That is correct, we are currently running make and bash to run cri-integration; we are reusing the same scripts that are already in place, so we wouldn't have to maintain 2 different scripts / building and running methods at the same time.